api/metadata/game_metadata.proto

```diff
@@ -37,6 +37,8 @@ message GameScore {
 message Goal {
     optional int32 frame_number = 1;
     optional api.PlayerId player_id = 2;
+    optional bool pre_rumble_items = 3;
```

Comment:
We might want to wrap this in a rumble message.
That way it doesn't it take space for non rumble games.

Reply:
how did you imagine this? Maybe have something like list of pre rumble goals inside GameMetadata?

Reply:
I was more imaginaging 
```
message Goal {
...
optional RumbleData rumble_data = 3;
}
```

(ofc using better naming than that)